src/edn.rs

```diff
@@ -40,3 +40,26 @@ fn test_field_to_keyword_uppercase() {
     );
     assert_eq!(field_to_keyword("TX___TX_TIME".to_string()), ":tx/tx-time");
 }
+
+fn camel_to_kebab(s: &str) -> String {
```

Comment:
looks nice! This sounds like a good function to export, maybe in `edn-rs`?

Reply:
Do you mean to replace the solution we have there for this? :thinking: